dev/tests/unit/Magento/FunctionalTestFramework/DataGenerator/Handlers/PersistedObjectHandlerTest.php

```diff
@@ -32,6 +32,30 @@ public function setUp()
         TestLoggingUtil::getInstance()->setMockLoggingUtil();
     }
 
+    public function testCreateEntityWithNonExistingName()
+    {
+        // Test Data and Variables
+        $entityName = "InvalidEntity";
+        $entityStepKey = "StepKey";
+        $scope = PersistedObjectHandler::TEST_SCOPE;
+
+        $exceptionMessage = "Entity \"" . $entityName . "\" does not exist." .
+            "\nException occurred parsing action at StepKey \"" . $entityStepKey . "\"";
```

Comment:
Will require change if exception msg changes

Reply:
changed `parsing` to `executing`